pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -429,6 +429,17 @@ RUN /opt/conda/bin/mamba install -y python=${PYTHON_VERSION} pytorch=${PYTORCH_V
   -c nvidia \
   && /opt/conda/bin/mamba clean -afy
 
+# Override only sm-pytorch
+RUN /opt/conda/bin/mamba install --force-reinstall -y \
+  pytorch=2.0.1=sm_py3.10_cuda11.8_cudnn8.9.5_nccl2.18.3_0_smp_2.0.0_pt_2.0.1 --override-channels \
+  -c https://sagemaker-distributed-model-parallel.s3.us-west-2.amazonaws.com/smp-2.0.0-pt-2.0.1/2023-12-11/smp-v2/ \
+  -c pytorch \
+  -c numba/label/dev \
+  -c pytorch-nightly \
+  -c nvidia \
+  -c conda-forge \
+  && /opt/conda/bin/mamba clean -afy
```

Comment:
Does the ofi package comes with this installation? if not, you can manually install from our conda channel. `mamba install aws-ofi-nccl --strict-channel-priority --override-channels -c https://aws-ml-conda.s3.us-west-2.amazonaws.com`